packages/vxrn/src/create.ts

```diff
@@ -518,6 +519,34 @@ export const create = async (optionsIn: VXRNConfig) => {
     defineEventHandler(() => `packager-status:running`)
   )
 
+  router.post(
+    '/symbolicate',
+    defineEventHandler(async (e) => {
+      const body = await readBody(e)
+      // React Native sends stack as JSON but tests content-type to text/plain, so
+      // we cannot use JSON schema to validate the body.
+
+      try {
+        const { stack } = JSON.parse(body as string) as {
+          stack: ReactNativeStackFrame[]
+        }
+        const platform = inferPlatformFromStack(stack)
+
+        if (!platform) {
+          return new Response('Cannot infer platform from stack trace', { status: 400 })
+        }
+
+        // console.info({ msg: 'Starting symbolication', platform, stack })
+        const results = await processStacks(stack)
+        // console.info(results)
+        return new Response(JSON.stringify(results))
```

Comment:
You can just `return results` as far i know, h3 will handle the response setting and other things.

Reply:
Oh, cool!